README.md

```diff
@@ -133,7 +140,70 @@ await autoModeTopic.publish();
 autoModeTopic.setValue('25 Ball Auto and Climb');
 ```
 
-### More info
+### Subscribing to Multiple Topics
+
+You can also subscribe to multiple topics by using a "wildcard" through creating a prefix topic.
+
+For example, here's a subscription for an Accelerometer with topics `/MyTable/Accelerometer/X`, `/MyTable/Accelerometer/Y`, and `/MyTable/Accelerometer/Z`:
+
+```typescript
+import { NetworkTables, NetworkTablesTypeInfos } from 'ntcore-ts-client';
+
+// Get or create the NT client instance
+const ntcore = NetworkTables.getInstanceByTeam(973);
+
+// Create the accelerator topic
+const accelerometerTopic = ntcore.createPrefixTopic('/MyTable/Accelerometer/');
+
+let x, y, z;
+
+// Subscribe to all topics under the prefix /MyTable/Accelerometer/
+accelerometerTopic.subscribe((value, params) => {
+  console.log(`Got Accelerometer Value: ${value} from topic ${params?.name}`); // i.e. Got Accelerometer Value: 9.81 from topic /MyTable/Accelerometer/Y
```

Comment:
Very minor:

Following you code, I believe the prefix subscribe callback will only be called if there is a `knownTopic`. So I think `params` in the prefix case should be able to be non-null, right?

If so, making it non-null for the callback will make it less confusing for any clients of the library and require less use of the `?`/optional chaining.

Reply:
This is a side-effect of the immediateNotify feature. I think I'll just make a breaking change removing it